modules/platforms/dotnet/Apache.Ignite.Core/Binary/BinaryBasicNameMapper.cs

```diff
@@ -45,6 +46,20 @@ public class BinaryBasicNameMapper : IBinaryNameMapper
         /// </summary>
         public bool IsSimpleName { get; set; }
 
+        /// <summary>
+        /// Domain to be added as a first part of Java type name.
+        /// Java and .Net assumes usage of different naming conventions.
+        /// Java package name first part is a domain: com, org, ru, etc. and second part is a company name: acme, company, etc.
+        /// .Net naming conventions don't use domain and start with a company name.
+        /// </summary>
+        public string JavaDomain { get; set; }
```

Comment:
I would prefer the `NamespacePrefix` name for this property - seems to be more technically correct, it is not necessarily Java-specific.

Reply:
OK. Fixed.